src/components/navbar.tsx

```diff
@@ -0,0 +1,262 @@
+                                    }}
+                                >
+                                    Subscription
+                                </Button>
+                            </CustomLink>
+                        </Menu>
+                    </Box>
+                    <AdbIcon
+                        sx={{ display: { xs: "flex", md: "none" }, mr: 1 }}
+                    />
+                    <Typography
+                        variant="h5"
+                        noWrap
+                        component="a"
+                        href=""
+                        sx={{
+                            mr: 2,
+                            display: { xs: "flex", md: "none" },
+                            flexGrow: 1,
+                            fontFamily: "monospace",
+                            fontWeight: 700,
+                            letterSpacing: ".3rem",
+                            color: "inherit",
+                            textDecoration: "none",
+                        }}
+                    >
+                        LOGO
+                    </Typography>
+                    <Box
+                        sx={{
+                            flexGrow: 1,
+                            display: { xs: "none", md: "flex" },
+                        }}
+                    >
+                        <CustomLink to="/login">
+                            <Button
+                                onClick={handleCloseNavMenu}
+                                sx={{
+                                    my: 2,
+                                    color: "white",
+                                    display: "block",
+                                }}
+                            >
+                                Login
+                            </Button>
+                        </CustomLink>
+                        <CustomLink to="/signup">
+                            <Button
+                                onClick={handleCloseNavMenu}
+                                sx={{
+                                    my: 2,
+                                    color: "white",
+                                    display: "block",
+                                }}
+                            >
+                                Sign Up
+                            </Button>
+                        </CustomLink>
+                        <CustomLink to="/subscription">
+                            <Button
+                                onClick={handleCloseNavMenu}
+                                sx={{
+                                    my: 2,
+                                    color: "white",
+                                    display: "block",
+                                }}
+                            >
+                                Subscription
+                            </Button>
+                        </CustomLink>
+                    </Box>
```

Comment:
Consider refactoring into a custom `styled.component` and `navigationItems.map => <.../>`  for better DRY

Reply:
Oh Absolutely, all of this will be deleted in my next PR I was just testing something out. But thank you! Haha